client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -86,6 +80,30 @@ var CheckoutThankYou = React.createClass( {
 			return <h1>{ this.translate( 'Thank you for your purchase!' ) }</h1>;
 		}
 	},
+	thankYouSubHeader: function() {
+		var productName = this.getSingleProductName(),
+			headerText;
+
+		if ( this.cartHasFreeTrial() && productName ) {
+			headerText = this.translate( 'We hope you enjoy ' + "%(productName)s. What's next? Take it for a spin!", {
```

Comment:
Super minor, but I would avoid string concatenation here in favor of using either one double-quote string literal or a single-quote string literal with an escaped apostrophe. It took me a while to figure out why both were used here. :sweat_smile: